packages/ember-flight-icons/package.json

```diff
@@ -16,75 +16,69 @@
   },
   "license": "MPL-2.0",
   "author": "HashiCorp Design Systems <design-systems@hashicorp.com>",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
   "scripts": {
-    "build": "ember build --environment=production",
+    "build": "rollup --config",
+    "prepublishOnly": "rollup --config",
+    "start": "rollup --config --watch",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
-    "start": "ember serve",
-    "test": "npm-run-all lint test:*",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
+    "lint:js:fix": "eslint . --fix"
   },
   "dependencies": {
+    "@embroider/addon-shim": "^1.8.7",
     "@hashicorp/flight-icons": "^2.24.0",
-    "ember-auto-import": "^2.6.3",
```

Comment:
i'm _pretty_ sure this is/was necessary for import logic within the addon to work correctly? or has v2 format changed things?

Reply:
the migration guidance [is suggesting we remove it](https://github.com/embroider-build/embroider/blob/main/docs/porting-addons-to-v2.md#part-4-convert-addon-to-v2) – happy to leave it there if we still need

Reply:
i will trust the guidance then 😄 . Just wanted to double check as testing this was pretty tricky if i recalled from past work on this.